### PR TITLE
More roundstart bloodbags

### PR DESCRIPTION
**Pull request by robosperg**
Behold my copy and paste skills

:cl:
 * rscadd: Medbay now starts with five bags of each bloodtype, up from one.

### CODE REVIEW

**Comment by amikawaii** [comment]
As much as i loathe depending on 3 cargo techs having a brain cell between them, 5 is a lot and it removes need for blood drive bot which I personally like.

**Comment by DrSnips** [comment]
Using something like `for(var/i = 0 to 4)` instead of that might be better.

File: code/game/machinery/kitchen/smartfridge.dm [hunk 1]
```diff
@@ -269,11 +269,36 @@
 	. = ..()
```
Comment (DrSnips) [inline code/game/machinery/kitchen/smartfridge.dm:270]: Like this.
```
for(var/i = 0 to 4)
	insert_item(new /obj/item/weapon/reagent_containers/blood/APlus(src))
	insert_item(new /obj/item/weapon/reagent_containers/blood/AMinus(src))
	etc```

**Comment by Kurfursten** [comment]
snips is right

File: code/game/machinery/kitchen/smartfridge.dm (hunk 1 above)
Comment (DrSnips) [reply code/game/machinery/kitchen/smartfridge.dm:270]: This would do it 5 times. Big brain coders start their lists at 0.

**Comment by Irradiation** [comment]
The ghost of "Fuck you Duny" is rolling in his grave.

**Comment by DrSnips** [comment]
Simply put it's a loop where it ticks up each time it's done with the stuff indented to the for(). In this case it would start at the first number and then stops after it gets to and finishes the second number.

File: code/game/machinery/kitchen/smartfridge.dm (hunk 1 above)
Comment (DrSnips) [reply code/game/machinery/kitchen/smartfridge.dm:270]: Resolved.

**Comment by boy2mantwicethefam** [comment]
I'm boy2mantwicethefam, and this is my favorite pull request on /vg/.

**Comment by ghost** [comment]
Why not make bloodbags have more blood instead?

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
If we're going with #22605 as well, I agree that increasing bloodbag capacity would be better. Maybe start with 2x of each bloodbag, and increase bloodbag capacity 2x.
Reason for this being that Medbay will go through bloodbags faster, and because the "meta" is still ordering 5 cargo bloodbag crates.

**Comment by robosperg** [comment]
cant find where blubb bags are in code, if someone can point it out I can make an alternative PR to increase size of bloodbags

Edit: This would only be useful if IVs stopped giving blood when people are full, the wild ride can get crazy and I would much rather have to hook a guy up to 2 IVs if needed rather than have a large amount of blood wasted away by larger bags

**Comment by DrSnips** [comment]
I'd rather have more of them rather than have more in them because if someone decides to run off to the shadow realm with one or two I'm no longer out of blood if cargo didn't do their thing.

**Comment by Kurfursten** [comment]
reagent_containers/blood_pack.dm

Dare I ask where you were looking for them if not in the reagent_containers folder?

**Comment by robosperg** [comment]
I was looking under /obj/item/weapon/, where is the reagant container folder?

**Comment by Kurfursten** [comment]
modules/reagents/reagent_containers

You can search for folder names (in windows, dream maker, and even here with the find file button), so try that in the future

**Review by ShiftyRail — APPROVED** [review]
The code works.

**Pull request merged**